less_users/models.py

```diff
@@ -1,3 +1,4 @@
+import datetime
 from datetime import date
```

Comment:
Dziwnie to wygląda. Wydaje mi się, że lepiej by było zrobić:
```python
from datetime import date, timedelta
```
albo samo `import datetime`

Reply:
tak, nie spojrzałam na górne importy, stąd zdublowane datetime - poprawiłam